client/src/pages/searches/Mine.js

```diff
@@ -0,0 +1,154 @@
+import { gql } from "@apollo/client"
+import { SEARCH_OBJECT_TYPES, setSearchQuery } from "actions"
+import API from "api"
+import ConfirmDestructive from "components/ConfirmDestructive"
+import Fieldset from "components/Fieldset"
+import Messages from "components/Messages"
+import {
+  jumpToTop,
+  mapPageDispatchersToProps,
+  PageDispatchersPropType,
+  useBoilerplate
+} from "components/Page"
+import SavedSearchTable from "components/SavedSearchTable"
+import { deserializeQueryParams } from "components/SearchFilters"
+import _isEmpty from "lodash/isEmpty"
+import PropTypes from "prop-types"
+import React, { useState } from "react"
+import { Button, ControlLabel, FormControl, FormGroup } from "react-bootstrap"
+import { connect } from "react-redux"
+import { useHistory } from "react-router-dom"
+import utils from "utils"
+
+const GQL_GET_SAVED_SEARCHES = gql`
+  query {
+    savedSearches: mySearches {
+      uuid
+      name
+      objectType
+      query
+    }
+  }
+`
+
+const GQL_DELETE_SAVED_SEARCH = gql`
+  mutation($uuid: String!) {
+    deleteSavedSearch(uuid: $uuid)
+  }
+`
+
+const Mine = ({ setSearchQuery, pageDispatchers }) => {
```

Comment:
Naming could be better. MySearches, MySavedSearches, etc... Mine is way too generic and does not give a hint about component behavior.